eslintrc-airbnb.js

```diff
@@ -0,0 +1,134 @@
+module.exports = {
+  root: true,
+  env: {
+    commonjs: true, // ConD-specific
+    es6: true
+  },
+  extends: ['airbnb'],
+  // Support language features that are provided by Babel (not required for JSX)
+  parser: "babel-eslint",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 2019
+  },
+  rules: {
+    "no-unused-vars": [
+      "error",
+      {
+        "argsIgnorePattern": "^_", // Allow unused arguments starting with an underscore
+        "varsIgnorePattern": "^_" // Allow unused variables starting with an underscore
+      }
+    ],
+
+    // No semicolons
+    "semi": ["error", "never"],
+    "no-unexpected-multiline": "error",
+
+    // 1. We follow the convention of naming "private" properties with a leading underscore
+    // 2. This conflicts with the no-unused-vars config
+    "no-underscore-dangle": "off",
+
+    // Could go either way on this one. Personally prefer without.
+    "comma-dangle": ["error", "never"],
+    
+    // Multiple variable declarations (without assignment) on the same line without multiple let/const
+    "one-var": ["error", { "initialized": "never", "uninitialized": "always" }],
+    "one-var-declaration-per-line": ["error", "initializations"],
+
+    // Maybe we want to enforce JSX files having the .jsx extension, but ConD hasn't so far
```

Comment:
I would not be opposed to this because it makes it very easy to look at a file name and know it's a react component vs opening the file or looking at the tree to discern what it is. Granted we often add view to the end of the filenames, but maybe if we use jsx extensions then view isnt so important

Reply:
I'd support enforcing the use of the `.jsx` extension. It seems like a pretty reasonable thing to add and assists editors in using the right language.